packages/framework/src/Resources/views/Admin/Content/Order/orderItem.html.twig

```diff
@@ -87,7 +107,19 @@
             </div>
         </td>
         <td class="text-right">-</td>
-        <td class="text-right">{{ self.priceWithVatWidget(orderPaymentForm.unitPriceWithVat) }}</td>
+
+        {% if inputPriceType is same as(constant('Shopsys\\FrameworkBundle\\Model\\Pricing\\PricingSetting::INPUT_PRICE_TYPE_WITH_VAT')) %}
+            <td class="text-right">
+                {{ self.inputPriceWidget(orderPaymentForm.unitPriceWithVat) }}
+            </td>
+            <td class="text-right">{{ self.calculablePriceWidget(orderPaymentForm.unitPriceWithoutVat) }}</td>
+        {% elseif inputPriceType is same as(constant('Shopsys\\FrameworkBundle\\Model\\Pricing\\PricingSetting::INPUT_PRICE_TYPE_WITHOUT_VAT')) %}
+            <td class="text-right">
+                {{ self.calculablePriceWidget(orderPaymentForm.unitPriceWithVat) }}
```

Comment:
unitPriceWithVat should be used here always?

Reply:
I do not understand the question. `calculablePriceWidget` and `inputPriceWidget` are changing by INPUT_PRICE_TYPE here.